_pytest/fixtures.py

```diff
@@ -864,11 +864,12 @@ def yield_fixture(scope="function", params=None, autouse=False, ids=None, name=N
     else:
         return FixtureFunctionMarker(scope, params, autouse, ids=ids, name=name)
 
-defaultfuncargprefixmarker = fixture()
-funcarg_prefix_warning = '{name}: declaring fixtures using "pytest_funcarg__" prefix is deprecated ' \
-                         'and scheduled to be removed in pytest 4.0.\n' \
-                         'remove the prefix and use the @pytest.fixture decorator instead'
 
+defaultfuncargprefixmarker = fixture()
+funcarg_prefix_warning = (
+    '{name}: declaring fixtures using "pytest_funcarg__" prefix is deprecated '
+    'and scheduled to be removed in pytest 4.0.\n'
+    'Please remove the prefix and use the @pytest.fixture decorator instead.')
```

Comment:
I've thought about removing the newline here, too: after all it would fail to be filtered with something like `… | grep -v '^WC'` currently.
What do you think?